addons/sale_project/static/tests/tours/sale_project_tour.js

```diff
@@ -0,0 +1,124 @@
+import { _t } from "@web/core/l10n/translation";
+import { registry } from "@web/core/registry";
+import { stepUtils } from "@web_tour/tour_service/tour_utils";
+import { markup } from "@odoo/owl";
+
+registry.category("web_tour.tours").add("task_create_sol_tour", {
+    test: true,
+    url: "/web",
+    rainbowManMessage: _t("Congratulations! You have successfully created a sales order item on the fly from a Task."),
+    steps: () => [stepUtils.showAppsMenuItem(), {
+    trigger: ".o_app[data-menu-xmlid='project.menu_main_pm']",
+    position: "bottom",
```

Comment:
Same comment than the previous tour, also I think `bottom` is the default value, no? 🧐